compute/zgbmm.c

```diff
@@ -65,7 +65,8 @@
  *          The width of the band below the diagonal in band matrix A.
  *
  * @param[in] ku
- *          The width of the band above the diagonal in band matrix A.
+ *          The width of the band [above/to the right of] the diagonal
```

Comment:
Perhaps I've got the way we'd like band matrices to be defined confused, but when nonsquare matrices come into play I find it makes more sense to think of the upper bandwidth as to the right of the band rather than the diagonal band. Consider a band matrix which is 5x10, ku=3. Though the diagonal only lives in the first 5 columns, there are elements who are still in the 'band' living in the 6th, 7th, and 8th columns. Perhaps the case of a nonsquare band matrix does not come up enough to make note of these cases.

As per your request, I'll use the word `bandwidth` though. I was hesitant to use it at first since there is this a true notion of `bandwidth` for the matrix (namely `1+ku+kl`), but what I wrote can certainly be improved upon.

Reply:
For the case I was thinking about, I think I agree with you again. I now imagine the diagonal stretching down beneath the matrix, and those elements in question still being 'above' the diagonal!